docs/configure.rst

```diff
@@ -0,0 +1,163 @@
+| Argument               | Description                                      |
++------------------------+--------------------------------------------------+
+| ``-h``,                | Print the help message, then exit.               |
+| ``--help``             |                                                  |
++------------------------+--------------------------------------------------+
+| ``-v``,                | Print the program's version number, then exit.   |
+| ``--version``          |                                                  |
++------------------------+--------------------------------------------------+
+
+For instance, if you want to look up the Croud help, simply run:
+
+.. code-block:: console
+
+  sh$ croud --help
+
+.. SEEALSO::
+
+  For a full documentation of Croud commands, check out the
+  `Croud CLI section`_.
+
+.. highlights:: console
+
+========================
+Sign in to CrateDB Cloud
+========================
+
+This page shows you how to sign in to CrateDB Cloud using `Croud`_.
+
+.. NOTE::
+
+  This document assumes that you have `created a CrateDB Cloud account`_.
+
+Run thew following command in your CLI to sign in to CrateDB Cloud:
+
+.. code-block:: console
+
+    sh$ croud login
+
+The CrateDB Cloud sign-in page will open in a new browser tab.
+
+.. cloud-sign-in-page.png
+.. image:: _assets/img/cloud-sign-up-page.png
+
+Provide your username and password and select *Sign in*.
+
+In another new browser tab, you should get the message:
+`You have successfully logged into CrateDB Cloud!`
+
+Close the tab and return to Croud.
+
+Verify that you are signed in by giving the ``croud`` command and passing in
+``me`` to retrieve the data of the user that is currently signed in:
+
+.. code-block:: console
+
+    sh$ croud me
+
+If you're signed in then CrateDB Cloud displays your email address and username,
+like so:
+
+.. code-block:: console
+
+   +--------------------+--------------+
+   | email              | username     |
+   |--------------------+--------------|
+   | youruser@email.com | yourusername |
+   +--------------------+--------------+
+
+In case you're not signed in, run the ``croud`` command and pass in ``login`` to
+sign in to CrateDB Cloud:
+
+.. code-block:: console
+
+  sh$ croud login
+
+Next steps
+==========
+
+You're now signed in to CrateDB Cloud. Let's
+`create an organization and project`_ to set you up for deploying your
+first `CrateDB cluster`_.
+
+.. _Croud: https://crate.io/docs/cloud/cli/en/latest/
+
+.. _created a CrateDB Cloud account: https://crate.io/docs/crate/cloud-getting-started/en/latest/first-steps/sign-in.html
+
+.. _create an organization and project: https://crate.io/docs/crate/cloud-getting-started/en/latest/first-steps/create.html
+
+.. _CrateDB cluster: https://crate.io/docs/crate/guide/en/latest/architecture/shared-nothing.html
+
+
+Next steps
+==========
+
+With the Croud CLI tool installed you can start using it by
+`signing in to CrateDB Cloud`_.
+
```

Comment:
double empty new lines before start of link block

please update this elsehwere too